src/config/mod.rs

```diff
@@ -5547,6 +5547,9 @@ mod tests {
             Some(default_cfg.coprocessor.region_split_size() * 3 / 4 / ReadableSize::kb(1));
         default_cfg.raft_store.region_split_check_diff =
             Some(default_cfg.coprocessor.region_split_size() / 16);
+        default_cfg
```

Comment:
Should optimize first and then query for its size.

Reply:
In this case it's same result.  But i agree with you.